pilot/docker/Dockerfile.proxyv2

```diff
@@ -17,6 +17,9 @@ ADD pilot-agent /usr/local/bin/pilot-agent
 ADD envoy_pilot.yaml.tmpl /etc/istio/proxy/envoy_pilot.yaml.tmpl
 ADD envoy_policy.yaml.tmpl /etc/istio/proxy/envoy_policy.yaml.tmpl
 ADD envoy_telemetry.yaml.tmpl /etc/istio/proxy/envoy_telemetry.yaml.tmpl
+
+ADD ca-root-cert.pem /etc/istio/certs/ca-root-cert.pem
```

Comment:
What is this ca-root-cert and why would we include it ???

Reply:
/hold

Reply:
root ca is now baked into proxy, wait for envoy's change to fetch root cert through sds, it could be removed(will remove this before merging to master)